src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs

```diff
@@ -269,9 +269,18 @@ internal static VsDebugTargetInfo4 GetDebuggerStruct4(IDebugLaunchSettings info)
             debugInfo.AppPackageLaunchInfo = info.AppPackageLaunchInfo;
             debugInfo.project = info.Project;
 
+            if (IsRemoteDebugging(info))
```

Comment:
The legacy project system does this based on when the app is "WinExe", why is this different? 

See CVsProjDebugLaunch.cpp:940, this value eventually flows down to the debug info.

Reply:
Or perhaps this bug was supposed to be against the legacy project system? @rajkumar42 was #3057 intended for legacy or new project system, or both?

Reply:
Ping?

Reply:
I need to setup a remote test environment to confirm.  I agree that this is not the right fix.

Reply:
@davkean apologies, I don't know the difference between legacy vs. the new project system. 'fSendToOutputWindow' was a relatively new feature - so I don't think it is in any project system yet (except for Docker).